topostats/default_config.yaml

```diff
@@ -53,6 +53,13 @@ grains:
     grain_crop_padding: 2 # Padding to apply to the grain crop bounding box
     upper_norm_bound: 5.0 # Upper bound for normalisation of input data. This should be slightly higher than the maximum desired / expected height of grains.
     lower_norm_bound: -1.0 # Lower bound for normalisation of input data. This should be slightly lower than the minimum desired / expected height of the background.
+  vetting:
```

Comment:
Would it be possible to include these descriptions in the `docs/config.md` please?

Reply:
Yep! Just wanted overall high-level feedback before adding finishing touches (to prevent doing the finishing touches twice if someone requested major changes)